alws/alembic/versions/444f9f1f0ac1_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
Can you please add a message when generating the migration? Right now it's empty and it'd be great to provide a message as we always do. Also, this message is added in the file name too, so it's easily identifiable. I guess you missed passing the  `-m` option when generating the migration.